src/Components/Server/src/Circuits/RemoteJSDataStream.cs

```diff
@@ -40,22 +40,20 @@ public static async Task<bool> ReceiveData(RemoteJSRuntime runtime, long streamI
             RemoteJSRuntime runtime,
             IJSStreamReference jsStreamReference,
             long totalLength,
-            long maximumIncomingBytes,
+            long signalRMaximumIncomingBytes,
```

Comment:
This change wasn't part of the API review, but I just felt this updated name could avoid future confusion (ex. clarifying it's not the max incoming bytes for the whole stream somehow).

Reply:
Great clarification. I probably *would* have thought it referred to the stream length before :)